subprojects/core/src/main/java/org/gradle/initialization/EvaluateSettingsBuildOperationType.java

```diff
@@ -22,6 +22,15 @@
 public class EvaluateSettingsBuildOperationType implements BuildOperationType<EvaluateSettingsBuildOperationType.Details, EvaluateSettingsBuildOperationType.Result> {
     @UsedByScanPlugin
     public interface Details {
+        /**
+         * The path of the build configuration that contains these projects.
```

Comment:
I don't think this comment is necessary, beyond the `@since` bit. We use `buildPath` on this things quite a bit.

Reply:
done